src/Microsoft.AspNetCore.Mvc.Razor/Internal/CompilerCacheResult.cs

```diff
@@ -17,29 +20,42 @@ public struct CompilerCacheResult
         /// Initializes a new instance of <see cref="CompilerCacheResult"/> with the specified
         /// <see cref="Compilation.CompilationResult"/>.
         /// </summary>
+        /// <param name="relativePath">Application relative path to the view file.</param>
```

Comment:
Either hyphenate "application-relative" or switch to the much more common `Path of the view file relative to the application base.`